### ISSUE TITLE
HACS Dataset: Action Recognition Model

### ISSUE THREAD

**Issue by ASMIftekhar** [frostinassiky/gtad#47]
Hello, 
Thanks for your excellent repository.

I see from 'data/hacs_annotations/hacs_val_cls_top2.json', you predicted top-2 actions on the validation set of HACS. Do you have the same json files for the testing and training sets of HACS? If you have that it would be nice if you can share that. Otherwise, can you point me to the repository that you used to predict the actions on HACS?

**Comment by frostinassiky** [comment]
Hello, @ASMIftekhar . Thanks for your interest in G-TAD. 

The action recognition model for HACS is [TSM](https://arxiv.org/abs/1811.08383). Please use this [repo](https://github.com/mit-han-lab/temporal-shift-module) to train the network.
